beacon-chain/core/blocks/block_operations.go

```diff
@@ -825,22 +802,8 @@ func VerifyIndexedAttestation(ctx context.Context, beaconState *stateTrie.Beacon
 			pubkeys = append(pubkeys, pk)
 		}
 	}
+	return attestationutil.VerifyIndexedAttestation(ctx, indexedAtt, pubkeys, beaconState.GenesisValidatorRoot(), beaconState.Fork())
```

Comment:
This takes in 5 arguments. Can we refactor it to be better?

Reply:
thanks. chnaged